Sprint-2/2-mandatory-debug/1.js

```diff
@@ -1,13 +1,21 @@
 // Predict and explain first...
-//  =============> write your prediction here
+//  =============> write your prediction here - undefined as we don't have parameter return
 
-function sum(a, b) {
+function sum1(a, b) {
   return;
   a + b;
 }
 
-console.log(`The sum of 10 and 32 is ${sum(10, 32)}`);
+console.log(`The sum of 10 and 32 is ${sum1(10, 32)}`);
+
+// =============> write your explanation here - undefined as we don't have parameter return
 
```

Comment:
Can you give a bit more of an explanation of what is structurally wrong with the code?